yourbench/pipeline/lighteval.py

```diff
@@ -87,65 +84,43 @@ def run(config: Dict[str, Any]) -> None:
         logger.info("lighteval stage is disabled. Skipping.")
         return
 
-    # ----------------------------------------
-    # 1) Identify subset names
-    # ----------------------------------------
-    single_shot_subset = stage_cfg.get("single_shot_subset", "single_shot_questions")
-    multi_hop_subset = stage_cfg.get("multi_hop_subset", "multi_hop_questions")
-    chunked_subset = stage_cfg.get("chunked_subset", "chunked_documents")
-    output_subset = stage_cfg.get("output_subset", "lighteval")
-
-    # We'll default to using the global dataset name from HF config if not specified
-    base_dataset_name = config.get("hf_configuration", {}).get(
-        "global_dataset_name", "yourbench_dataset"
-    )
-    output_dataset_name = stage_cfg.get("output_dataset_name", base_dataset_name)
-
-    logger.info(
-        f"Starting lighteval stage. single_shot_subset='{single_shot_subset}', "
-        f"multi_hop_subset='{multi_hop_subset}', chunked_subset='{chunked_subset}', "
-        f"output_subset='{output_subset}'"
-    )
+    logger.info("Starting lighteval stage")
```

Comment:
I would not call this stage "lighteval", maybe "saving lighteval compatible dataset"